### PR TITLE
295: GitHub Action

### PR DESCRIPTION
**Pull request by wwelling**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -56,3 +56,13 @@ jobs:
           folder: static/weaver-components
           clean: true
           single_commit: true
+
+      - name: Cleanup xvfb
```
Comment (wwelling) [inline .github/workflows/build.yml:60]: I do not think this is needed. Also, we can use official coverapps action. We are using the forked branch for affording json coveralls. We intend to switch back to the official action when this [PR](https://github.com/coverallsapp/github-action/pull/46) is merged. We do not need the fork for this repo.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -39,7 +39,7 @@ jobs:
         run: xvfb-run --auto-servernum npm run test:ci
 
       - name: "Coveralls Coverage Report"
-        uses: coverallsapp/github-action@master
+        uses: MikeEdgar/github-action@raw_coverage_file
```
Comment (wwelling) [inline .github/workflows/build.yml:42]: We can use official coverapps action. We are using the forked branch for affording json coveralls. We intend to switch back to the official action when this PR is merged. We do not need the fork for this repo.

File: .github/workflows/build.yml (hunk 1 above)
Comment (rmathew1011) [reply .github/workflows/build.yml]: Removed cleanup xvfb block

File: .github/workflows/build.yml (hunk 2 above)
Comment (rmathew1011) [reply .github/workflows/build.yml]: Switched to the official coveralls usage - instead of MikeEdgars.

**Review by jeremythuff — APPROVED** [review]

**Pull request merged**